app/styles/_ember-version-graphic.scss

```diff
@@ -3,9 +3,9 @@
   $white-color: #FFFFFF;
 
   $mascots: (
-    tomster-deprecated: 'https://emberjs.com/images/tomsters/deprecated-24565384.png',
-    tomster-lts: 'https://emberjs.com/images/tomsters/lts-8974c71c.png',
-    zoey: 'https://emberjs.com/images/tomsters/zoey-2f531686.png'
+    tomster-deprecated: '',
```

Comment:
Where are these?

Reply:
yeah this was just a quick hack to remove them because the links to those images are broken and it was showing up as an error in the browser console. i'm not sure if they were intentionally broken, but I think if they worked they'd show some version of the tomsters in the version cards. not sure how that fits with the new design.

Reply:
It's probably because of the fingerprinting. I'd try it without that and see if it works?

Reply:
💯